### ISSUE TITLE
An error occured: (0 , _uuid.default) is not a function

### ISSUE THREAD

**Issue by abt86croz** [rdunk/sanity-plugin-table#9]
Unable to use Tables | keep getting a UID error, also the syntax for the docs is wrong (no curly braces around the Table object). How can I mitigate this issue?

```javascript
import { ImTable as icon } from 'react-icons/im';

export default {
  name: 'tables',
  title: 'Tables',
  type: 'document',
  icon,
  fields: [
    {
      type: 'string',
      name: 'title',
      validation: (Rule) => Rule.required(),
    },
    {
      name: 'tables',
      title: 'ABT Tales',
      type: 'table',
    },
  ],
};
```

<img width="376" alt="Screen Shot 2020-12-20 at 7 53 19 PM" src="https://user-images.githubusercontent.com/36944810/102729240-e91bbb80-42fd-11eb-8c70-5de659c252f7.png">

**Comment by alejandrovrojas** [comment]
It seems the `uuid` import remains to be destructured: `import { uuid } from '@sanity/uuid';`, which would fix the issue.

https://github.com/rdunk/sanity-plugin-table/blob/faa2b3f4678af43e6fa5d1d4995fb02e34efef47/src/component.js#L1

**Comment by rdunk** [comment]
Thanks for your patience. This is fixed in #12.
